client/components/Layout/Layout.tsx

```diff
@@ -43,22 +43,26 @@ const Layout = (props: Props) => {
 
 		if (collection) {
 			return {
-				type: 'collection' as const,
+				event: 'collection' as const,
```

Comment:
"type" is now the type of event, which gets sent automatically and is either "pageView" or "track"

"event" now specifies what kind of "pageView" or "track" event is sent